src/components/common/PageHeaderWrapper.tsx

```diff
@@ -9,7 +9,7 @@ import useDebounce from '../../utils/useDebounce';
 
 const TOP_BAR_HEIGHT = 64;
 const DEBOUNCE_TIME = 500;
-const LEFT_NAV_WIDTH = 220;
+export const LEFT_NAV_WIDTH = 200;
```

Comment:
This is to match designs, might change if design comes back and says the designs are incorrect